x-pack/test/functional/services/machine_learning/job_wizard_common.ts

```diff
@@ -7,10 +7,12 @@ import expect from '@kbn/expect';
 import { ProvidedType } from '@kbn/test/types/ftr';
 
 import { FtrProviderContext } from '../../ftr_provider_context';
+import { MachineLearningCommonProvider } from './common';
 import { MachineLearningCustomUrlsProvider } from './custom_urls';
 
 export function MachineLearningJobWizardCommonProvider(
   { getService }: FtrProviderContext,
+  mlCommon: ProvidedType<typeof MachineLearningCommonProvider>,
```

Comment:
In the future, I usually like to export this type from the module containing the provider so I can just import the `MlCommon` type without having to use these long types everywhere.

Reply:
Good idea! I've created a PR for this: #52612